operator/charts/embedded-cluster-operator/templates/embedded-cluster-lam-service-config.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: embedded-cluster-lam-service-config
+  labels:
+    troubleshoot.sh/kind: support-bundle
+    {{- with (include "embedded-cluster-operator.labels" $ | fromYaml) }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+data:
+  support-bundle-spec: |
+    apiVersion: troubleshoot.sh/v1beta2
+    kind: SupportBundle
+    metadata:
+      name: embedded-cluster-lam-service-config
+      labels:
+        troubleshoot.sh/kind: support-bundle
+    spec:
+      collectors:
+        - runDaemonSet:
+            name: "local-artifact-mirror-service-config"
+            namespace: embedded-cluster
+            podSpec:
+              containers:
+              - image: {{ .Values.utilsImage }}
+                imagePullPolicy: Always
+                args: ["chroot","/host","cat","/etc/systemd/system/local-artifact-mirror.service.d/embedded-cluster.conf"]
```

Comment:
would it be possible to collect all files in this directory?

Reply:
Something like `"/host","sh", "-c", "cat /etc/systemd/system/local-artifact-mirror.service.d/*"` will work. All files would be concatenated into one file. That might be confusing. I think its better to collect one file at a time